Bottom: 889a2cce9d07af3e53a803f98987824775164192
Top:    5432ac67408f3825825d4bde2a0f8df2f77fa276
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-23 17:44:08 +0000

Refresh of hid-gtk-gui-drc-window-c-refac

---

diff --git a/src/hid/gtk/gui-drc-window.c b/src/hid/gtk/gui-drc-window.c
index f612184..5d4bb35 100644
--- a/src/hid/gtk/gui-drc-window.c
+++ b/src/hid/gtk/gui-drc-window.c
@@ -31,6 +31,7 @@
 #include "error.h"
 #include "search.h"
 #include "draw.h"
+#include "find.h"
 #include "pcb-printf.h"
 #include "undo.h"
 #include "set.h"
